src/main/scala/wdlTools/types/Unification.scala

```diff
@@ -188,6 +184,7 @@ case class Unification(conf: Options) {
   //    T_Var(0) -> T_Int
   //    T_Var(1) -> T_String
   //
+  @throws(classOf[TypeUnificationException])
```

Comment:
Please don't use decorators. The next person to modify this code is going to start asking himself if he should add it everywhere, and if so, how can he figure out if this needs to be reported.